docs/writing-tests.md

```diff
@@ -300,7 +314,13 @@ it('some test', async ({ browser }) => {
 Parameters:
 
 - state (required) `String` – state name; should be unique within one test
-- selector (required) `String|String[]` – DOM-node selector that you need to capture
+- selector (optional) `String|String[]` – DOM-node selector that you need to capture. If not specified or skipped, will be set to `body` and the following options will be automatically added to opts: 
+```{
+    allowViewportOverflow: true,
+    compositeImage: false,
+    captureElementFromTop: false
+}``` 
+So, assertView without `selector` parameter will take a screenshot of the current viewport.
 - opts (optional) `Object`:
```

Comment:
For assert viewport it is second argument. This is not clear in the current description.